src/gateways/getTypeLabelByType.js

```diff
@@ -14,7 +14,7 @@ export const getTypeLabelByType = type => {
     }
 
     if (type === 'http' || !type) {
-        return i18n.t('Generic')
+        return i18n.t('Send as header')
```

Comment:
Is this what we've communicated so far to the user?

Reply:
Not sure about that, but this is the wording that was requested in the requirements doc.

Reply:
I'm not 100% certain, but I think this is regarding the `Is header` checkbox:

![image](https://user-images.githubusercontent.com/10101831/91857468-3c430300-ec68-11ea-9de7-77f5cf05ff09.png)

Reply:
Ah yeah. Hmm yeah that seems plausible. I'll remove those changes.

Reply:
I already thought it sounded a bit weird. Removed.